gui/frontend/src/dashboard/dashboard.coffee

```diff
@@ -161,6 +161,7 @@ angular.module "mindbenderApp.dashboard", [
     $scope.title = "Snapshot " + $routeParams.snapshotId
     $scope.loading = false
     $scope.hideLoader = true
+    $scope.tableTab = { active: false }
```

Comment:
Any reason using an object here?  It seems active is the only flag here.  It's fine if you envision there will be more fields being added to this.